fedimintd/src/bin/main.rs

```diff
@@ -114,11 +153,6 @@ async fn main() -> anyhow::Result<()> {
     let key = get_key(opts.password, salt_path)?;
     let cfg = read_server_configs(&key, opts.data_dir.clone())?;
 
-    let local_task_set = tokio::task::LocalSet::new();
```

Comment:
I'm not sure what the `local_task_set` is good for. Doesn't seem to get used.

Reply:
Oh damn. This has something to do with `consensus` not being `Send` or something, and the fact that we need to `spawn_local` it. This was just creating some set of threads for it to run in a non-send mode(?), then "enter" it, and then check the results from it. Bleh.

Reply:
The thing is, it was never used. The only side effect was some guard that was spawned. I removed it and it didn't seem any different, but I did have to run the future in a `spawn_local` anyway.

Reply:
Lol, it does have side effects …